website/blog/2020-12-15-certificate-pinning.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Certificate Pinning Case Study"
+author: "Radhakrishnan Asokan"
+---
+
+A case study on certificate pinning.
+
+<!--truncate-->
+
+### Introduction
+
+In HTTPS handshake, the server responds with a public certificate issued by Certificate Authority (CA) to establish a connection. If CA or Root certificate was compromised and issues a certificate without the domain owner's consent, the client may face the man-in-the-middle (MTM) attack. To protect users from this vulnerability, an app can employ SSL pinning.
+
+![MITM](/learn/assets/blog_certificate_pinning/mitmdiagram.png)
+
+Certificate Pinning is an additional layer of security which protects communication between client and server. The Standard HTTPS verifies whether the connection is secure but it cannot verify whether you are communicating with the actual server or an intercepted server.
+
+## What Certificate Pinning offers
+
+Standard HTTPS checks only whether the connection is secure. Along with the HTTPS, we use certificate pinning which will verify whether you are communicating with the actual server.
```

Comment:
this is incorrect. HTTPS also checks the certificate

Reply:
Made changes to the sentences, Please review again.